llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -17341,6 +17341,85 @@ static SDValue combineScalarCTPOPToVCPOP(SDNode *N, SelectionDAG &DAG,
   return DAG.getZExtOrTrunc(Pop, DL, VT);
 }
 
+static SDValue combineSHL(SDNode *N, TargetLowering::DAGCombinerInfo &DCI,
```

Comment:
The naming convention in this file would have this routine named performSHLCombine

Reply:
Renamed, thanks!